### PR TITLE
incus: update to 6.2

### PR DESCRIPTION
**Pull request by dkwo**
Tested.

### CODE REVIEW

**Comment by dkwo** [comment]
to do: add a readme.void mentioning incus groups, subpackages and perhaps
`echo "root:1000000:1000000000" | doas tee -a /etc/subuid /etc/subgid`
and maybe also set CGROUP_MODE in rc.conf to unified instead of hybrid.

**Comment by dkwo** [comment]
@ahesford can you take a look at this one as well?

**Comment by sbromberger** [comment]
Hi all,

6.2 was just released - any way we might just bump the version here?

**Review by ahesford — CHANGES_REQUESTED** [review]
I don't have any experience with incus, but I've made a few suggestions to improve the readme and variable handling in the template.

**Comment by dkwo** [comment]
Thanks!  Implemented the suggestions and updated to 6.2

File: srcpkgs/incus/files/README.voidlinux [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+and possibly the incus-tools one.
+Membership in the _incus-admin and _incus groups is also needed to
+interact with the corresponding services, which need to be started.
+
+Similar to lxc/lxd, something like
+
+echo "root:1000000:1000000000" | doas tee -a /etc/subuid /etc/subgid
+
+should be setup.  One may also want to set CGROUP_MODE in rc.conf to
+unified instead of hybrid.
```
[the remark this answers is not in the document]
Comment (dkwo) [reply srcpkgs/incus/files/README.voidlinux]: done.

**Comment by dkwo** [comment]
Still failing on 32 bit. I thought that patch had been merged in v6.2
is anyone actually using incus on those archs? I wonder if it's worth it supporting them..

**Comment by dkwo** [comment]
the patch fixes 32 bit. i'll leave this as draft until it is merged upstream.

**Comment by dkwo** [comment]
patch now merged upstream.

**Comment by sbromberger** [comment]
Thank you both!

**Pull request merged**